app/src/main/java/ru/prometeydev/movie/FragmentMoviesDetails.kt

```diff
@@ -60,23 +76,71 @@ class FragmentMoviesDetails : Fragment() {
     override fun onStart() {
         super.onStart()
 
-        loadData()
+        loadActors()
     }
 
     override fun onDetach() {
         recycler = null
+        movie = null
 
         super.onDetach()
     }
 
+    // загрузка данных
     private fun loadData() {
+        arguments?.let {
+            movie = Movie(
+                id = it.getLong(MOVIE_ID),
+                name = it.getString(MOVIE_NAME, ""),
+                genre = it.getString(MOVIE_GENRE, ""),
+                rating = it.getFloat(RATING),
+                reviewsCount = it.getInt(REVIEWS_COUNT),
+                ageLimit = it.getInt(AGE_LIMIT),
+                filmCoverDrawable = it.getInt(COVER),
+
+                additional = MovieAdditional(
+                    pictureDrawable =  it.getInt(MOVIE_PICTURE),
+                    storyLine = it.getString(STORY_LINE, ""),
+                    actors = it.getParcelableArrayList<Actor>(ACTORS_LIST)?.let { actors -> actors } ?: arrayListOf()
+                )
+            )
+        }
+    }
+
+    private fun loadActors() {
         (recycler?.adapter as? ActorsAdapter)?.apply {
-            bindActors(ActorsDataSource().getActors())
+            bindActors(movie?.additional!!.actors)
         }
     }
 
     companion object {
-        fun instance() = FragmentMoviesDetails()
+
+        fun instance(movie: Movie) = FragmentMoviesDetails().apply {
+            arguments = Bundle().apply {
+                putLong(MOVIE_ID, movie.id)
```

Comment:
This is not best solution - you can just pass movie id and get specific movie from your data source!

Reply:
You can also make movie as parcelable but I don't this it is also the best solution

Reply:
I'll fix it, I'll do it when transferring only the MovieId